ouroboros-consensus/src/Ouroboros/Consensus/Node/ProtocolInfo/Byron.hs

```diff
@@ -87,19 +92,13 @@ protocolInfoByron (NumCoreNodes numCoreNodes) (CoreNodeId nid) params gc =
       , pInfoInitState  = ()
       }
   where
+    proofOfCredentials :: CoreNodeId -> PbftLeaderCredentials -> PBftIsLeader PBftCardanoCrypto
+    proofOfCredentials nid' (PbftLeaderCredentials sk dlg) =
+      PBftIsLeader
+      { pbftCoreNodeId = nid'
```

Comment:
One of the next steps will be to compute this based on the genesis verification key (as in my original WIP/hacking branch).

Reply:
Good point, I had the same thought on Slack!